internal/byok/command.go

```diff
@@ -13,12 +13,17 @@ type command struct {
 }
 
 type out struct {
-	Id        string   `human:"ID" serialized:"id"`
-	Key       string   `human:"Key" serialized:"key"`
-	Roles     []string `human:"Roles" serialized:"roles"`
-	Cloud     string   `human:"Cloud" serialized:"cloud"`
-	State     string   `human:"State" serialized:"state"`
-	CreatedAt string   `human:"Created At" serialized:"created_at"`
+	Id                string   `human:"ID" serialized:"id"`
+	DisplayName       string   `human:"Display Name,omitempty" serialized:"display_name,omitempty"`
+	Key               string   `human:"Key" serialized:"key"`
+	Roles             []string `human:"Roles" serialized:"roles"`
+	Cloud             string   `human:"Cloud" serialized:"cloud"`
+	ValidationRegion  string   `human:"Region,omitempty" serialized:"region,omitempty"`
+	State             string   `human:"State" serialized:"state"`
+	CreatedAt         string   `human:"Created At" serialized:"created_at"`
+	ValidationPhase   string   `human:"Validation Phase" serialized:"validation_phase"`
+	ValidationMessage string   `human:"Message,omitempty" serialized:"message,omitempty"`
+	ValidationSince   string   `human:"Since" serialized:"since"`
```

Comment:
```suggestion
	ValidationMessage string   `human:"Validation Message,omitempty" serialized:"validation_message,omitempty"`
	ValidatedSince   string   `human:"Validated Since" serialized:"validated_since"`
```
Let’s update this to make it more intuitive for users, the current format `| Since            | 2022-11-12 08:30:00 +0000 UTC` might be a bit confusing

Reply:
`Validated Since` may be misleading, as the field represents since when a field is in the given validation phase (e.g., consecutively failed validation since X. Wdyt?